librz/include/rz_util/rz_str_search.h

```diff
@@ -0,0 +1,36 @@
+#ifndef RZ_STR_SEARCH_H
+#define RZ_STR_SEARCH_H
+
+#include <rz_util/rz_str.h>
+#include <rz_util/rz_assert.h>
+#include <rz_util/rz_buf.h>
+#include <rz_list.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef struct {
+	char *string;
+	ut64 addr;
+	ut32 size; // size of buffer containing the string in bytes
+	ut32 length; // length of string in chars
+	RzStrEnc type;
+} RzDetectedString;
+
+typedef struct {
+	size_t buf_size;
+	size_t max_uni_blocks;
+	size_t min_str_length;
+} RzUtilStrScanOptions;
```

Comment:
I think there's no need for this to be here. It seems you are just using this inside str_search.c, so you could probably just move this structure in that file.

Reply:
now RzUtilStrScanOptions is a parameter of `rz_scan_strings`